drivers/media/i2c/imx500.c

```diff
@@ -3029,6 +3427,12 @@ static int imx500_init_controls(struct imx500 *imx500)
 		v4l2_ctrl_new_custom(ctrl_hdlr, &network_fw_fd, NULL);
 	imx500->device_id =
 		v4l2_ctrl_new_custom(ctrl_hdlr, &cam_get_device_id, NULL);
+	imx500->enable_injection =
+		v4l2_ctrl_new_custom(ctrl_hdlr, &enable_injection_ctrl, NULL);
+	imx500->input_tensor_fd =
+		v4l2_ctrl_new_custom(ctrl_hdlr, &input_tensor_fd, NULL);
+	imx500->injection_cmp_frm =
+		v4l2_ctrl_new_custom(ctrl_hdlr, &injection_cmp_frm, NULL);
```

Comment:
Added 3 controls, but not increased the hint passed in `ret = v4l2_ctrl_handler_init(ctrl_hdlr, 16);` at line 3374.

I make it 17 controls total (two added by `v4l2_ctrl_new_fwnode_properties`), which implies that the number was wrong before.